modules/virt-temporary-token-VNC.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-consoles.adoc
+
+:_content-type: PROCEDURE
+[id="virt-temporary-token-VNC_{context}"]
+= Generating a temporary token for the VNC console
+
+Generate a temporary token to access the VNC of a virtual machine (VM).
+
+.Prerequisites
+
+* A running virtual machine with {VirtProductName} 4.14 or later and xref:../../virt/about-virt/virt-architecture#virt-about-ssp-operator_virt-architecture[`ssp-operator`] 4.14 or later
+
+.Procedure
+
+. Enable the feature gate in the HyperConverged (`HCO`) custom resource (CR):
++
+[source,yaml]
+----
+$ oc patch hyperconverged kubevirt-hyperconverged -n {CNVNamespace} \
+ --type json -p '[{"op": "replace", "path": \
```

Comment:
The `\` at the end of lines should not be there. The whole patch is enclosed in single quotes `'`, so the `\` would be present in the final command, and that would make it incorrect.

Reply:
@akrejcir Would the following work?

```
$ oc patch hyperconverged kubevirt-hyperconverged -n {CNVNamespace} \
 --type json -p '[{"op": "replace", "path": ' \
 '"/spec/featureGates/deployVmConsoleProxy", "value": true}]'
```

Reply:
No, because this would be interpreted as:
```bash
oc patch ...  -p '[{"op": "replace", "path": '  '"/spec/featureGates/deployVmConsoleProxy", "value": true}]'
```

The patch argument `-p` is broken into 2 parts separated by space:
```
' [{"op": "replace", "path": '
' "/spec/featureGates/deployVmConsoleProxy", "value": true}] '
```
but only the first part is passed as `-p` argument. The other part will probably cause an error.

Reply:
Got it -thanks!